apps/chameleon/app/(profile)/profile/edit/_components/portfolio-image.tsx

```diff
@@ -0,0 +1,27 @@
+'use client'
+import { Portfolio } from '@prisma/client'
```

Comment:
Typically, naming conflicts wouldn't pose an issue. However, since your component shares its name, "Portfolio", with a variable, the build process encounters a scoping dilemma and struggles to differentiate between the two when they are named identically.

To resolve this issue, you can explicitly import "Portfolio" as a type from '@prisma/client' by using the following syntax:

```typescript
import type { Portfolio } from '@prisma/client';
```

This approach clarifies to the TypeScript compiler that "Portfolio" is being imported strictly for type-checking purposes, effectively resolving the naming conflict without altering the runtime behavior of your component.